org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/client/proxy/ErrorDoRestClientExceptionTransformer.java

```diff
@@ -58,6 +58,12 @@ protected RuntimeException transformByResponseStatusFamily(Response.Status.Famil
   protected RuntimeException transformClientError(RuntimeException e, Response response, BiFunction<String, RuntimeException, VetoException> vetoExceptionFactory) {
     return safeTransformEntityErrorResponse(e, response, () -> {
       ErrorDo error = response.readEntity(ErrorResponse.class).getError();
+      if (error == null) {
+        // ErrorDo might be missing, if a ServletFilter aborts the request with an e.g. forbidden http status code.
+        // The servlet container might then return a generic json response which does not contain a Scout ErrorDo.
+        // Such a response could look like: {"servlet":"ServletName","message":"Forbidden","url":"/api/","status":"403"}
+        return vetoExceptionFactory.apply(e == null ? null : e.getMessage(), e);
```

Comment:
Suggestion: Use statusInfo reason phrase if no message available (like in fallback part)